Cargo.toml

```diff
@@ -36,6 +37,7 @@ serde = { version = "1.0.99", default-features = false, optional = true }
 pure-rust-locales = { version = "0.5.2", optional = true }
 criterion = { version = "0.4.0", optional = true }
 rkyv = {version = "0.7", optional = true}
+bytecheck = { version = "~0.6.8", optional = true, default-features = false }
```

Comment:
If this is intended to only allow 0.6.x versions, we can leave off the `~` as the `^` operator can be left implicit in Cargo manifests, and it appears that `~` is equivalent to `^` for the major-0 versions.